src/css/vars.css

```diff
@@ -0,0 +1,25 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --box-sizing: border-box;
+  --primary-color: #060b35;
+  --secondary-color: #616070;
+  --theme-color: #00acdc;
+  --white-color: #fff;
+  --card-border: 1px solid #f3f3f3;
+  --font-family: "Roboto", "Arial", "Helvetica", "sans-serif";
+  --line-height-lg: 18px;
+  --line-height-md: 16px;
+  --line-height-sm: 14px;
+  --font-weight-bold: 700;
+  --font-weight-medium: 500;
+  --font-weight-light: 400;
+  --font-size-lg: 16px;
+  --font-size-md: 14px;
+  --font-size-sm: 12px;
+  --font-size-xs: 10px;
+  --star-size: 16px;
+  --star-distance: 4px;
+  --star-bg-inactive: url("../images/star.svg") center no-repeat;
+  --star-bg-active: url("../images/star-active.svg") center no-repeat;
```

Comment:
You dont need to use variables for all values, like `--box-sizing` , `--font-family` etc. Only for important value, or value which use many times.